db/models/model.js

```diff
@@ -1,34 +1,60 @@
 const db = require("../connection");
-// const { articleData } = require("../data/test-data");
 
 const fetchTopicsFromDatabase = () => {
   return new Promise((resolve, reject) => {
-    db.query("SELECT * FROM topics;")
-      .then((queryResult) => {
-        resolve(queryResult.rows);
-      })
-      .catch((error) => {
-        reject(error);
-      });
+    db.query("SELECT * FROM topics;").then((queryResult) => {
+      resolve(queryResult.rows);
+    });
   });
 };
 
+function fetchAllArticlesFromDataBase() {
+  return new Promise((resolve, reject) => {
```

Comment:
We don't need to create a new promise here. PSQL will return a resolved or rejected promise for us